Microsoft.Toolkit/Extensions/StringExtensions.cs

```diff
@@ -195,5 +195,42 @@ public static string Truncate(this string value, int length, bool ellipsis)
 
             return value ?? string.Empty;
         }
+
+        /// <summary>
+        /// Translate numeric file size in bytes to a human-readable shorter string format.
+        /// </summary>
+        /// <param name="size">file size in bytes.</param>
+        /// <returns>Returns file size short string.</returns>
+        public static string ToFileSizeString(this long size)
```

Comment:
This is not a string extension, it's a long extension, and I think it should be a "convert method", not an extension method to long. It's too broad.

Reply:
Fixed moved to a new `Convert` static class.